mysite/staticfiles/css/globalstyle.css

```diff
@@ -17,3 +18,9 @@
     overflow: hidden;
     margin: 10px 0px;
 }
+
+.bg-light{ 
```

Comment:
klassen bg-light är en klass från bootstrap och ska inte ändras av oss. skapa hellre en ny egen klass och skippa att använda bg-light i elementet. 

Denna css-filen läses in efter bootstrap och ärvs på alla sidor i CreativeStash. Denna ändringen gör så alla element som använder bg-ligt får denna färgen